src/hotspot/share/gc/shenandoah/shenandoahEvacTracker.cpp

```diff
@@ -121,7 +134,8 @@ class ShenandoahStatAggregator : public ThreadClosure {
 };
 
 ShenandoahCycleStats ShenandoahEvacuationTracker::flush_cycle_to_global() {
-  ShenandoahEvacuationStats mutators, workers;
+  bool gen_mode = ShenandoahHeap::heap()->mode()->is_generational();
```

Comment:
Why not use the `_generational` member here?

Reply:
Done.